community/kernel/src/test/java/org/neo4j/test/assertion/Assert.java

```diff
@@ -41,6 +41,23 @@ private Assert()
     {
     }
 
+    public static <E extends Exception> void assertException( ThrowingAction<E> f, Class typeOfException ) throws
```

Comment:
Yes, but that method is annoying if the exception message is `null`. If you do not care about the message, why should a called have to decide whether to but "" or null as the message argument.

Fixed the implementation to reuse the previous one though.